README.md

```diff
@@ -60,29 +60,29 @@ This will work with `notepad++.exe`, however some functionality in Notepad++ wil
 
 To build SciLexer.dll without PCRE support (for both 32-bit & 64-bit):
 
- 1. For 32-bit, open a command prompt *for building* ([a.k.a. the *Developer Command Prompt for VS2013*](https://msdn.microsoft.com/en-us/library/f2ccy3wt.aspx))
-    - From the IDE, you can do this by right clicking on a file in Solution Explorer,
-      and clicking "Open Command Prompt". This will open up a command prompt with all the proper environment variables.
+ 1. For 32-bit Windows, open a command prompt *for building* ([a.k.a. the *Developer Command Prompt for VS2013*](https://msdn.microsoft.com/en-us/library/f2ccy3wt.aspx))

```

Comment:
This change introduces ambiguity. It's the SciLexer.dll that’s going to be 32-bits, (and therefore Notepad++), but not necessarily Windows.
I recommend changing this to "To build a 32-bits DLL, ..." or leaving this as it was.